workflow/schemas/resources.schema.yaml

```diff
@@ -78,6 +78,25 @@ properties:
         type: string
         description: partition to use on the cluster for 'samtools_index'
 
+  samtools_sort:
```

Comment:
Another suggestion I have, shouldn't we sort this level of stanzas, keeping default_resources highest up, of course?

Reply:
please elaborate, I don't really understand

Reply:
Sorry, I meant `samtools_sort` and `bam_merge` and so on, shouldn't they be sorted in the schemas like so:
```
default_resources:
   ...
a_rule:
  ...
bam_merge:
  ...
samtools_sort:
  ...
vep:
  ...
```
just an super general simplified example of course.

Reply:
good suggestion!